csm/SystemMan.py

```diff
@@ -0,0 +1,139 @@
+import CommsMan as CommsMan
+import StorageMan as StorageMan
+import MotorMan as MotorMan
+import CameraMan as CameraMan
+import PerceptionMan as PerceptionMan
+
+from CommsMan import CommsMan
+from StorageMan import StorageMan
+from MotorMan import MotorMan
+from CameraMan import CameraMan
+from PerceptionMan import PerceptionMan
+
+# from ImageSources import ImageSource, LocalVideo, StillImage, Camera
+# from PerceptionUtils import BoundingBox
+
+import queue
+import threading
+import time
+import sys
+
+class SystemMan():
+
+    def __init__(self):
+        self.running = True
+        self.inFrame = False
+        self.currVideo = 0
+
+        # Instantiate Subsystems
+        self.per = PerceptionMan(threshold=0.5)
+        self.cam = CameraMan("../testData/testOutput.mp4")
+        self.mot = MotorMan()
+        
+
+        self.sto = StorageMan()
+        self.comQ, self.comT = queue.Queue(maxsize=1), queue.Queue(maxsize=1)
+        self.com = CommsMan(self.comQ, self.comT)
+
+    def sendMessageToRemote(self, message):
```

Comment:
I don't really think we need these one line helper methods, but I'm open to hearing why you think we would

Reply:
This is to provide an interface for us when we want to interact directly with SystemMan without removing the abstraction of specific data structures/objects used. Maybe more importantly, limiting any interaction with SystemMan to take place through these interfaces will help if SystemMan's implementation were to change.

Reply:
Ah I see. Abstraction... I like it.